runtime/src/accounts_db.rs

```diff
@@ -753,6 +776,66 @@ impl AccountsDB {
         (reclaims, dead_keys)
     }
 
+    // Construct a vec of the keys needed for cleaning from uncleaned_keys and
+    // zero_lamport_keys sets.
```

Comment:
Maybe add comment like this?

```rust
// Such a vec of the pubkeys is a combined set of all of alive zero lamport pubkeys
// and all of pubkeys updated since last clean.
```